### PR TITLE
fix(ui/dfn-list): close popup only if link is clicked

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: src/ui/dfn-list.js [hunk 1]
```diff
@@ -18,6 +18,9 @@ ul.classList.add("respec-dfn-list");
 const render = hyperHTML.bind(ul);
 
 ul.addEventListener("click", ev => {
+  if (ev.target instanceof HTMLElement && !ev.target.matches("a")) {
```
Comment (saschanaz) [inline src/ui/dfn-list.js]: This potentially closes the ui when the target is not HTMLElement. It won't happen, but then maybe we don't need that specific condition?

Comment (sidvishnoi) [reply src/ui/dfn-list.js]: Typescript was crying eventTarget has no matches()
Thought this was better than ts-ignore
Open to other ways

Comment (saschanaz) [reply src/ui/dfn-list.js]: Oh. How about `if (ev.target instanceof HTMLElement && ev.target.matches("a")) {` and wrapping the code then? Becasue reading this line makes me think whether `Element` is intentionally allowed or not 😂

Comment (sidvishnoi) [reply src/ui/dfn-list.js]: That's cool by me. Updating

**Pull request merged**